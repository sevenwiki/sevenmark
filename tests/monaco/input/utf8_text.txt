í•œêµ­ì–´
æ—¥æœ¬èªž
Emoji ðŸš€ test
Special chars: Ã¥Ã¤Ã¶