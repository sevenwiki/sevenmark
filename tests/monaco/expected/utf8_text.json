[
  {
    "Text": {
      "content": "í•œêµ­ì–´",
      "location": {
        "end_column": 10,
        "end_line": 1,
        "start_column": 1,
        "start_line": 1
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 2,
        "start_column": 10,
        "start_line": 1
      }
    }
  },
  {
    "Text": {
      "content": "æ—¥æœ¬èªž",
      "location": {
        "end_column": 10,
        "end_line": 2,
        "start_column": 1,
        "start_line": 2
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 3,
        "start_column": 10,
        "start_line": 2
      }
    }
  },
  {
    "Text": {
      "content": "Emoji ðŸš€ test",
      "location": {
        "end_column": 16,
        "end_line": 3,
        "start_column": 1,
        "start_line": 3
      }
    }
  },
  {
    "Text": {
      "content": "\n",
      "location": {
        "end_column": 1,
        "end_line": 4,
        "start_column": 16,
        "start_line": 3
      }
    }
  },
  {
    "Text": {
      "content": "Special chars: Ã¥Ã¤Ã¶",
      "location": {
        "end_column": 22,
        "end_line": 4,
        "start_column": 1,
        "start_line": 4
      }
    }
  }
]